common/scala/src/main/scala/whisk/spi/SpiLoader.scala

```diff
@@ -65,15 +72,20 @@ object SpiLoader {
     }
 }
 
-/**
- * Lookup the classname for the SPI impl based on a key in the provided Config
- */
+/** Lookup the classname for the SPI impl based on a key in the provided Config */
 object TypesafeConfigClassResolver extends SpiClassResolver {
-    val config = ConfigFactory.load()
+    private val config = ConfigFactory.load()
+
     override def getClassnameForKey(key: String): String = config.getString(key)
     override def getKeyForType[T](implicit man: Manifest[T]): String = "whisk.spi." + man.runtimeClass.getSimpleName
 }
 
+/**
+ * Object containing arbitrary objects acting as dependencies.
+ *
+ * This is solely a helper type to cross the border between possibly heterogeneous Spi
```

Comment:
Should "heterogeneous Spi interfaces" be "heterogeneous SpiFactoy implementations"?